test/mailers/previews/user_mailer_preview.rb

```diff
@@ -1,4 +1,20 @@
 # frozen_string_literal: true
 
 class UserMailerPreview < ActionMailer::Preview
+  # Preview this email at
+  # http://localhost:3000/rails/mailers/user_mailer/password_reset
+  def password_reset
+    user = User.first
+    user.reset_token = User.new_token
+    url = "http://localhost:3000" + "/password_resets/" + user.reset_token + "/edit?email=" + user.email
```

Comment:
Avoid the use of localhost, specially for testing environments, unless you really want to use localhost for something, like a database.
It is a common practice to use something that is more evident to be a mock-up, like example.com

Reply:
That would make sense. Done